include/zephyr/drivers/sensor.h

```diff
@@ -315,6 +315,8 @@ enum sensor_attribute {
 	SENSOR_ATTR_CALIBRATION,
 	/** Enable/disable sensor features */
 	SENSOR_ATTR_FEATURE_MASK,
+	/** Enable/disable continuous mode. */
+	SENSOR_ATTR_CONTINUOUS_MODE,
```

Comment:
I think that adding this feature in sensor.h requires a separate commit

Reply:
I have added this "Continuous Mode" to facilitate the power domain handling in the case of keeping the sensor active for a larger period of time - as such it is directly part of my functionality of introducing power domain to the driver. This change has nothing to do with fifo-mode and my functionality depends on this attribute being defined. Please let me know how to proceed in this case :)

Reply:
the fifo part is a different story.
But regarding the "Continuos Mode" I think that anyway the change to sensor.h must be for generic use, hence in a separate commit and well documented, as they might be used by other drivers as well

Reply:
Change now kept in a separate commit